submissions/zhenyakornilov/js-dom/js/index.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+
+import museums from "./data.js";
+import toggleDarkClass from "./utils.js";
+
+const themeSwitcher = document.getElementById("switcher");
+const listMenu = document.getElementById("menu");
+const dropdownBtn = document.getElementById("dropdown-button");
+const contentBody = document.getElementById("content");
+
+function renderButtons(museumsObj, menu) {
+  let buttonsHTML = museumsObj
+    .map(({ btnName }) => {
+      return `
+      <li class="nav-item">
+        <button class="btn-text">${btnName}</button>
+      </li>
+      `;
+    })
+    .join("");
+
+  menu.innerHTML = buttonsHTML;
+}
+
+function showMuseumInfo({ target }) {
+  if (target.closest(".btn-text")) {
+    let currentSelectedBtn = target.closest(".nav-item");
+    if (!currentSelectedBtn.classList.contains("active")) {
+      const [currentContentObject] = museums.filter(
+        ({ btnName }) => btnName === target.textContent
```

Comment:
Better use `id` to identify the target item, not the text. For example, we can have two items with the same button name. So, in such cases we should always use id for DOM manipulation